systemvm/debian/opt/cloud/bin/setup/patchsystemvm.sh

```diff
@@ -37,6 +37,8 @@ patch_systemvm() {
       echo "Restored keystore file and certs using backup" >> $logfile
    fi
    rm -fr $backupfolder
+   # Import global cacerts into 'cloud' service's keystore
+   keytool -importkeystore -srckeystore /etc/ssl/certs/java/cacerts -destkeystore /usr/local/cloud/systemvm/certs/realhostip.keystore -srcstorepass changeit -deststorepass vmops.com -noprompt || true
```

Comment:
Why importing the Java default cacerts? I mean, why not using it directly?
I see that you are already modifying it to add the Let's encrypt public key.

Also, why do you need the `|| true` at the end?

Reply:
For historic reasons, the keystore used by agent is at /usr/local/cloud/systemvm/cert in systemvm. This code/change ensures that the global ca cert (java) is imported into that local keystore. The reason `|| true` is used to ensure that this does not fail patching in case the keystore already has certs imported (say an already patched systemvm stopped and started).

The letsencrypt cacert is imported during systemvmtemplate building, the change in patchsystemvm.sh will ensure that we import cacerts during systemvm patching.

Reply:
ok, got it.